stable/spinnaker/templates/statefulsets/halyard.yaml

```diff
@@ -0,0 +1,110 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "spinnaker.fullname" . }}-halyard
+  labels:
+{{ include "spinnaker.standard-labels" . | indent 4 }}
+    component: halyard
+spec:
+  ports:
+  - port: 8064
+    name: daemon
+  clusterIP: None
+  selector:
+    app: {{ template "spinnaker.fullname" . }}
+    component: halyard
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ template "spinnaker.fullname" . }}-halyard
+  labels:
+{{ include "spinnaker.standard-labels" . | indent 4 }}
+spec:
+  serviceName: {{ template "spinnaker.fullname" . }}-halyard
+  replicas: 1
+  selector:
+    matchLabels:
+{{ include "spinnaker.standard-selector-labels" . | indent 6 }}
+      component: halyard
+  template:
+    metadata:
+      labels:
+{{ include "spinnaker.standard-labels" . | indent 8 }}
+        component: halyard
+    spec:
+      serviceAccountName: {{ template "spinnaker.fullname" . }}-halyard
```

Comment:
Could you update the chart to use our RBAC best practices?
https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/rbac.md

Reply:
Updated. PTAL